src/benchmarkstt/output/core.py

```diff
@@ -48,4 +48,8 @@ def result(self, title, result):
             print(',')
         self._line += 1
         print('\t', end='')
+
+        if isinstance(result, tuple) and hasattr(result, '_asdict'):
+            result = result._asdict()
```

Comment:
On the current master branch, the output of `--diffcounts` is an array:
```
~ $ benchmarkstt -r 'hello world' -rt argument -h 'hello werld' -ht argument --wer --diffcounts -o json
[
        {"title": "wer", "result": 0.5},
        {"title": "diffcounts", "result": [1, 1, 0, 0]}
]
```
But (I assume it's this line that I'm commenting on) this PR seems to turn it into a dictionary instead:
```
~ $ benchmarkstt -r 'hello world' -rt argument -h 'hello werld' -ht argument --wer --diffcounts -o json
[
        {"title": "wer", "result": 0.5},
        {"title": "diffcounts", "result": {"equal": 1, "replace": 1, "insert": 0, "delete": 0}}
]
```
This is really handy, but it looks like this feature doesn't have a unit test associated with it. Is it covered in the BDDs?

Reply:
You are right, neither is unit tested. I should add one before merging this PR